Maestro/subscriptions.json

```diff
@@ -721,6 +727,16 @@
         "vsoSourceBranch": "master"
       }
     },
+    // Trigger official build of the dotnet-docker samples when the source is changed
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/dotnet-docker/blob/master/samples/**/*"
```

Comment:
Is the `dotnet-docker` web hook set up? (https://github.com/dotnet/core-eng/blob/master/Documentation/Maestro/web-hooks.md)

Reply:
I just added it.